questions-data.js

```diff
@@ -0,0 +1,1303 @@
+    region: 'America'
+  },
+  {
+    expansion: 'CAHe3',
+    id: 246,
+    numAnswers: 2,
+    official: true,
+    text: 'When you get right down to it, _ is just _.',
+    region: 'America'
+  },
+  {
+    expansion: 'CAHe3',
+    id: 247,
+    numAnswers: 1,
+    official: true,
+    text: 'Turns out that _-Man was neither the hero we needed nor wanted.',
+    region: 'America'
+  },
+  {
+    expansion: 'CAHe3',
+    id: 248,
+    numAnswers: 1,
+    official: true,
+    text: 'As part of his daily regimen, Anderson Cooper sets aside 15 minutes for _.',
+    region: 'America'
+  },
+  {
+    expansion: 'CAHe3',
+    id: 249,
+    numAnswers: 1,
+    official: true,
+    text: "Money can't buy me love, but it can buy me _.",
+    region: 'America'
+  },
+  {
+    expansion: 'CAHe3',
+    id: 250,
+    numAnswers: 2,
+    official: true,
+    text: 'With enough time and pressure, _ will turn into _.',
+    region: 'America'
+  },
+  {
+    expansion: 'CAHe3',
+    id: 251,
+    numAnswers: 1,
+    official: true,
+    text: 'And what did you bring for show and tell?',
+    region: 'America'
+  },
+  {
+    expansion: 'CAHe3',
+    id: 252,
+    numAnswers: 1,
+    official: true,
+    text: 'During high school, I never really fit in until I found _ club.',
+    region: 'America'
+  },
+  {
+    expansion: 'CAHe3',
+    id: 253,
+    numAnswers: 1,
+    official: true,
+    text: "Hey, baby, come back to my place and I'll show you _.",
+    region: 'America'
+  },
+  {
+    expansion: 'CAHe3',
+    id: 254,
+    numAnswers: 2,
+    official: true,
+    text: "After months of practice with _, I think I'm finally ready for _.",
+    region: 'America'
+  },
+  {
+    expansion: 'CAHe3',
+    id: 255,
+    numAnswers: 1,
+    official: true,
+    text: 'To prepare for his upcoming role, Daniel Day-Lewis immersed himself in the world of _.',
+    region: 'America'
+  },
+  {
+    expansion: 'CAHe3',
+    id: 256,
+    numAnswers: 1,
+    official: true,
+    text: 'Finally! A service that delivers _ right to your door.',
+    region: 'America'
+  },
+  {
+    expansion: 'CAHe3',
+    id: 257,
+    numAnswers: 1,
+    official: true,
+    text: 'My gym teacher got fired for adding _ to the obstacle course.',
+    region: 'America'
+  },
+  {
+    expansion: 'CAHe3',
+    id: 258,
+    numAnswers: 2,
+    official: true,
+    text: 'Having problems with _? Try _!',
+    region: 'America'
+  },
+  {
+    expansion: 'CAHe3',
+    id: 259,
+    numAnswers: 1,
+    official: true,
+    text: "As part of his contract, Prince won't perform without _ in his dressing room.",
+    region: 'America'
+  },
+  {
+    expansion: 'CAHe3',
+    id: 260,
+    numAnswers: 2,
+    official: true,
+    text: "Listen, son. If you want to get involved with _, I won't stop you. Just steer clear of _.",
+    region: 'America'
+  }
+];
+
+module.exports = questions;
+
```

Comment:
Do you think moving this file and the "answers-data.js" file to a folder called say `seed_data` be a better way to organise our seed data files?

Reply:
Good idea!. It helps improve file organisation especially if the need arises for us to add more seed data files. I'll move the files into the suggested folder. Thanks.